imports/plugins/included/payments-example/client/settings/containers/exampleSettingsFormContainer.js

```diff
@@ -0,0 +1,93 @@
+import { composeWithTracker } from "/lib/api/compose";
+import { ExampleSettingsForm } from "../components";
+import { Meteor } from "meteor/meteor";
+import { Packages } from "/lib/collections";
+import { Loading } from "/imports/plugins/core/ui/client/components";
+import { TranslationProvider } from "/imports/plugins/core/ui/client/providers";
+
+import Alert from "sweetalert2";
+import { Reaction, i18next } from "/client/api";
+
+import React, { Component, PropTypes } from "react";
+
+class ExampleSettingsFormContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      apiKey: ""
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.saveUpdate = this.saveUpdate.bind(this);
+  }
+
+  handleChange(e) {
+    e.preventDefault();
+    this.setState({ apiKey: e.target.value });
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+
+    const packageId = this.props.packageData._id;
+    const settingsKey = this.props.packageData.registry[0].settingsKey;
```

Comment:
@aaronjudd the output of that method is an object in this case ""packageData" and the only way to access the settingsKey is by indexing like I did above. On autoforms you could just use {{settingsKey}} and get it.